scoring.js

```diff
@@ -0,0 +1,94 @@
+function qbScoring(qbStats) {
+  let passingYardScore = qbStats.passing.yards / 25
+  let passingTouchdownScore = qbStats.passing.touchdowns * 6
```

Comment:
all looks good. one suggestion is to create and user variables also for point values ( 6, -3, and 10)

because in future if we said, change all 6 to 3 for touchdown score.  youll have to search each one instead of just updating the variable.